dev_install.adoc

```diff
@@ -16,6 +16,15 @@ Configure authentication for the cluster using one of the following options:
 Option #1: Contact a member of the Dev Sandbox Team for instructions on how to configure the cluster to use our internal Dev SSO. +
 Option #2: Configure your own Keycloak server and set up authentication on the OpenShift cluster: https://docs.openshift.com/container-platform/4.6/authentication/configuring-internal-oauth.html
 
+
+Option #3: Deploy and configure keycloak internally as part of the cluster. Just add `DEV_SSO=true` parameter to the `dev` targets. For eg.: `make dev-deploy-latest DEV_SSO=true` will deploy latest version of the operators with a preconfigured keycloak instance.
```

Comment:
I know that it's the default user & password is printed at the end of the command, but I didn't notice it at the first glance. You can mention it here as well.
```suggestion
Option #3: Deploy and configure keycloak internally as part of the cluster. Just add `DEV_SSO=true` parameter to the `dev` targets. For eg.: `make dev-deploy-latest DEV_SSO=true` will deploy latest version of the operators with a preconfigured keycloak instance and one default keycloak user `user1@user.us` with password `user1`.
```

Reply:
thanks a lot for the feedback! makes sense!